packages/docs/pages/Typography/TypographyPage.tsx

```diff
@@ -79,6 +79,12 @@ export default () => (
         <Text italic>This text is italic.</Text>
         <Text strikethrough>This text is strikethrough.</Text>
         <Text underline>This color is underline.</Text>
+        <Text uppercase>This text is uppercase.</Text>
+        <Text lowercase>This text is lowercase.</Text>
+        <Text capitalize>This text is capitalize.</Text>
```

Comment:
```suggestion
        <Text capitalize>This text is capitalized.</Text>
```

Reply:
Should we change all the other ones (underline, etc) 🤔? I was just using the css value.

Reply:
No I was just thinking about proper grammar on this one.